tests/updateCards.spec.js

```diff
@@ -0,0 +1,102 @@
+import { test, expect } from '@playwright/test';
+const  credentials = require('../cypress/fixtures/credentials1.json');
+const { chromium } = require('@playwright/test');
+const { LoginPage } = require('./pages/loginPage.page');
+const { WorkspaceApi } = require('./pages/workspaceApi');
+const { BoardsApi } = require('./pages/boardsApi');
+const { CardsUi } = require('./pages/cardsUi')
```

Comment:
In the future when creating Page Object classes, it is better to just name them like: `CardsPage` or `NavPage`